docker/README.md

```diff
@@ -117,20 +106,20 @@ If your project uses packages that this image doesn't provide you might want to:
 1. create own Dockerfile that holds:
     ```dockerfile
     # Point at any base image that you find suitable to extend.
-    FROM emscripten/emsdk:1.38.25
+    FROM emscripten/emsdk:1.39.17
 
     # Install required tools that are useful for your project i.e. ninja-build
     RUN apt update && apt install -y ninja-build
 
     ```
 2. build it
-    ```shell
+    ```bash
```

Comment:
Does this make a difference anywhere?

Reply:
It doesn't make any difference, `bash` is an alias for `shell` within GitHub flavored markdown. I think @trzecieu added this to align it with the other code blocks.